### PR TITLE
Add in rod NamedChunk for pendulum

### PR DESCRIPTION
**Pull request by Ant13731**
Contributes to #2398

### CODE REVIEW

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: code/drasil-example/Drasil/DblPendulum/Body.hs [hunk 1]
```diff
@@ -175,7 +176,7 @@ tMods = [accelerationTM, velocityTM, newtonSL, newtonSLR]
 
 physSystParts :: [Sentence]
 physSystParts = map foldlSent [
-  [S "The rod"],
+  [S "The", phrase rod],
```
[the remark this answers is not in the document]
Comment (Ant13731) [reply code/drasil-example/Drasil/DblPendulum/Body.hs:179]: This is the only thing that has worked so far:
```Haskell
physSystParts = map foldlSent [[phraseNP (the rod)], [phraseNP (the mass)]]
```

[the remark this answers is not in the document]
Comment (Ant13731) [reply code/drasil-example/Drasil/DblPendulum/Body.hs:179]: I just realized that foldlSent is probably the problem.

**Comment by Ant13731** [comment]
The new `physSysParts` function also removes the periods from the list (see the html/tex files for what I mean), which I think looks better anyway.

**Comment by JacquesCarette** [comment]
We need to figure out where the period came from. Perhaps `foldlSent` did that?

**Comment by Ant13731** [comment]
Yes, `foldlSent` added the period at the end.

**Comment by JacquesCarette** [comment]
I'm sure there's a combinator for terminating sentences. You most definitely should not be doing it "by hand".

**Comment by Ant13731** [comment]
The `+:+.` was the only combinator I could find that terminates sentences with a period. It takes two sentences, so `EmptyS` must be the first argument for it to work. When I'm changing `Utils.Drasil` later on, would it be good to add a combinator that takes one `Sentence` and just appends a period to it (that is, if I don't find an existing one)?

**Comment by JacquesCarette** [comment]
Indeed, there is none.  Well, there is the private `wrap` in https://github.com/JacquesCarette/Drasil/blob/acf6d8dbd9632e14954e8e5027768edfd8ceb7d8/code/drasil-utils/Utils/Drasil/Fold.hs#L68-L76 which kind of does it, but it is not accessible, and is overkill.

I would be tempted to call it `:.` and used as a postfix operator - https://downloads.haskell.org/~ghc/8.8.4/docs/html/users_guide/glasgow_exts.html#postfix-operators.

**Review by JacquesCarette — APPROVED** [review]
I'll approve as is. A further PR can add the new combinator and fix this at the same time.
```
find drasil-example/ -type f -name '*.hs' -exec grep "\.\"" \{\} \; -print
```
finds a number of hard-coded periods in sentences that should be examined with this in mind.

**Comment by Ant13731** [comment]
I'm having a little bit of trouble incorporating `:.` as an operator. I looked it up and I think the `:` is reserved for data/type constructors, so implementing it as a function would be impossible. I also tried to use the postfixoperator language but I keep getting parse errors when trying to implement it. Is there an example of other postfix operators I could take a look at?
```
Parse error in pattern: (+.)
|
| ((+.) a) = a :+: S "."
|
```

**Comment by JacquesCarette** [comment]
You did enable the `PostfixOperators` option in the compile options, right?

Indeed, I used bad syntax. The documentation above mentions left sections, documented at https://www.haskell.org/onlinereport/exps.html#sections .  One then needs to follow up into the lexical structure...

Cutting that short: `!.` or `$.` or `&.` should all work.  I think I prefer `!.`.

```haskell
(!.) a = a :+: S "."
```
**should** work.

**Pull request merged**